cloudnet-common/src/main/java/de/dytanic/cloudnet/common/concurrent/ITask.java

```diff
@@ -57,8 +57,6 @@ public void onCancelled(ITask<V> task) {
 
     Collection<ITaskListener<V>> getListeners();
 
-    Callable<V> getCallable();
```

Comment:
Any reason to remove this? Would cause breaking changes.

Reply:
Well, no, not really, forgot to remove this change. I will add this later